File: Basic java codes/Prime.java
class Prime
{
 public static void main(String[] s)
 {
  int n=14,i,flag=1;
  for(i=2;i<=n/2;i++)
  {
   if(n%i==0)
   {
    flag=1;
    break;
   }
   else
   flag=0;
  }
 if(flag==1)
 System.out.println("number is not prime");
 else
 System.out.println("number is prime");
 }
}

File: Basic c codes/prime.c
#include<stdio.h>

int main()
{
    int num,flag=0;
    printf("Enter the number to check whether it is prime or not- ");
    scanf("%d",&num);
    for(int i=2;i<=(num/2)+1;i++)
    {
        if(num%i==0)
        {
            printf("Number is not prime.");
            flag=1;
            break;
        }
    }
    if(flag==0)
        printf("Number is prime.");
}
